perl-package/test.sh

```diff
@@ -0,0 +1,16 @@
+MXNET_HOME=$1
```

Comment:
Since you set `ws('workspace/ut-perl-cpu')`, that will actually be where the mxnet home is. If you change that I can merge and you can keep an eye on it. Thanks for waiting.